File: backend/Domain/Entities/ApplicationLegalRecord.cs
namespace Domain.Entities
{
    public class ApplicationLegalRecord
    {
       public int id { get; set; }
       public int id_application { get; set; }
       public int id_legalrecord { get; set; }
       public int id_legalact { get; set; }
       public DateTime created_at { get; set; }
       public DateTime updated_at { get; set; }
       public int created_by { get; set; }
       public int updated_by { get; set; }
    }
}

File: frontend/src/constants/application_legal_record.ts
import { Dayjs } from "dayjs";

export type application_legal_record = {
  
  id: number;
  id_application: number;
  id_legalrecord: number;
  id_legalact: number;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
};
